src/lae_site/handlers/__init__.py

```diff
@@ -8,14 +8,15 @@
 from twisted.python.filepath import FilePath
 
 from lae_site.handlers.web import JinjaHandler
-from lae_site.handlers.submit_subscription import SubmitSubscriptionHandler
+# TODO: Rename all handlers teh same way for consistency
```

Comment:
Typo `teh`

Reply:
this TODO is actually done, so will remove the comment